ruby_on_rails-postgres/web/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ruby:2.6.5
+
+# install nodejs / libdev / postgres-client
+RUN apt-get update -qq && apt-get install -y nodejs libpq-dev postgresql-client
+
+## install yarn
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
+    && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
+    && apt-get update && apt-get install -y yarn && rm -rf /var/lib/apt/lists/*
+
+RUN mkdir /myapp
+WORKDIR /myapp
+COPY Gemfile /myapp/Gemfile
+COPY Gemfile.lock /myapp/Gemfile.lock
+
+##
+RUN gem install bundler:2.0.2
+RUN bundle install
```

Comment:
You can use can do `RUN bundle check || bundle install` to avoid installing the gems on every build

https://bundler.io/man/bundle-check.1.html#NAME

Reply:
Good, I don't know this.